src/TestCase.php

```diff
@@ -198,4 +198,26 @@ protected function getEnvironmentSetUp($app)
     {
         // Define your environment setup.
     }
+
+    /**
+     * Migrate Laravel's default migrations.
+     * 
+     * @param  string $database
+     * 
+     * @return void
+     */
+    public function runLaravelDefaultMigrations($database = null)
+    {
+        $options = ['--path' => '../fixture/migrations'];
```

Comment:
The commit for this and the file renaming are on my branch if you need them.
As long as the file isn't renamed, this package's tests will fail.